pkg/asset/installconfig/platform.go

```diff
@@ -70,15 +71,15 @@ func (a *Platform) queryUserForPlatform() string {
 func (a *Platform) awsPlatform() (*asset.State, error) {
 	return assetStateForStringContents(
 		AWSPlatformType,
-		asset.QueryUser(a.InputReader, "Region:"),
+		asset.QueryUser(a.InputReader, "Region:", "us-east-1"),
```

Comment:
Ideally we'd pull this default from the user's config if and only if they selected the AWS platform.  Maybe replace your default strings with a `type default func() (default []byte, err error)` and have `QueryUser` call that helper to get the default value if the function is non-`nil`?  That way you could call `aws configure get region` in a sub-process if and only if we needed to prompt the user for a region.

Reply:
> The region is only asked if the user entered AWS anyway right now. Are you saying that we should pick up the default region from another config file (~/.awscreds etc)?

I think invoking `aws configure get region` in a sub-process is probably more reliable than writing our own reader for environment variables and `~/.aws`.  But I don't want to invoke `aws` if the user asks for (say) a libvirt cluster.  Hence my default-getting-callback suggestion.